backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/SaveVmExternalDataCommand.java

```diff
@@ -47,8 +46,8 @@ private boolean hasTpmDevice() {
         return !vmDeviceDao.getVmDeviceByVmIdAndType(getParameters().getVmId(), VmDeviceGeneralType.TPM).isEmpty();
     }
 
-    private boolean hasSecureBoot() {
-        return getVm().getBiosType() == BiosType.Q35_SECURE_BOOT
+    private boolean isUEFI() {
```

Comment:
A small naming convention suggestion: `isUefi` (similar to `isOvmf` and others).

Reply:
Ok, done. Please look at this.